src/query-builder/DeleteQueryBuilder.ts

```diff
@@ -144,16 +146,31 @@ export class DeleteQueryBuilder<Entity> extends QueryBuilder<Entity> {
         return this;
     }
 
+    /**
+     * Optional returning/output clause.
+     */
+    returning(returning: string): this {
+        this.expressionMap.returning = returning;
+        return this;
+    }
+
```

Comment:
probably we can add `output` method as well (to be friendly to sqlserver users) like this:

```ts
 output(output: string): this {
     return this.returning(output);
}
```